lib/core/presentation/widgets/common/collapsible/collapsible_section_widget.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:app/theme/theme.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+
+class CollapsibleSection extends StatefulWidget {
+  final String title;
+  final List<Widget> children;
+
+  CollapsibleSection({required this.title, required this.children});
+
+  @override
+  _CollapsibleSectionState createState() => _CollapsibleSectionState();
+}
+
+class _CollapsibleSectionState extends State<CollapsibleSection> {
+  bool _isExpanded = true;
+
+  @override
+  Widget build(BuildContext context) {
+    final colorScheme = Theme.of(context).colorScheme;
+    return Theme(
+        data:
+            lemonadeAppDarkThemeData.copyWith(dividerColor: Colors.transparent),
```

Comment:
I think we should not use fixed dark theme here, let use themeData from context and copywith with a new value